src/redux/books/booksSlice.js

```diff
@@ -1,46 +1,98 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
+const appId = 'q8FdluzHjZBBWXH5Y85r';
```

Comment:
[OTPIONAL] Why not use environment variables 🤔 : Instead of hardcoding the `appId`, you could use environment variables to store it. This will make it easier to switch between different environments (e.g., development, staging, production) without changing the code. 🌎